### PR TITLE
Add: Traditional Chinese README.md.

### PR DESCRIPTION
**Pull request by RejectVanity**
## Changes

Create a Traditional Chinese README, specify the file name for the Simplified Chinese README, and store both in the "READMEs" directory.

(Standardize the filename of the Simplified Chinese README according to the IETF language tag standard.)

## Why 

Write here why you think this should be merged

## Checkmarks

- [x] The modified functions have been tested.
- [x] Used the same indentation as the rest of the project.
- [x] Updated documentation (changelog).

## Additional information

I accidentally closed pull request #81 recently. Now, I have created a new pull request (#84) and also synchronized the latest changes from #80 into it.

### CODE REVIEW

**Review by ThePedroo — APPROVED** [review]
LGTM!

File: READMEs/README_zh-TW.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+雖然 Zygisk Next 的開發者在 Android 社群中享有盛譽，並且備受信任，但這並不代表他們的專案就完全沒有任何惡意程式碼或漏洞。我們（PerformanC）理解他們因某些原因選擇保持閉源，但我們堅信開源才是更好的選擇。
+
+## 優勢
+
+- 永遠是自由及開放原始碼軟體（FOSS）
+
+## 依賴項
+
+| 工具            | 說明                            |
```
Comment (ThePedroo) [inline READMEs/README_zh-TW.md:25]: The `|` must be aligned

File: READMEs/README_zh-TW.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+|-----------------|----------------------------------------|
+| `Android NDK`   | Android 原生開發工具包     |
```
Comment (ThePedroo) [inline READMEs/README_zh-TW.md:27]: Ditto

File: READMEs/README_zh-TW.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+### C++ 依賴項
+
+| 依賴 | 說明                   |
```
Comment (ThePedroo) [inline READMEs/README_zh-TW.md:31]: Ditto

File: READMEs/README_zh-TW.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+|------------|-------------------------------|
+| `lsplt`    | Android 的簡單 PLT（程式連結表） 勾取   |
```
Comment (ThePedroo) [inline READMEs/README_zh-TW.md:33]: Ditto

**Review by ThePedroo — CHANGES_REQUESTED** [review]
Actually, I missed something. Feel free to fix those in the Simplified Chinese too.

**Review by RainyXeon — COMMENTED** [review]
Idk if it's possible to be aligned or not.
But imo, I prefer the `|` to be aligned in README_zh-TW

Let me know it's not possible

File: READMEs/README_zh-TW.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+| 工具            | 說明                                   |
+|-----------------|---------------------------------------|
+| `Android NDK`   | Android 原生開發工具包                  |
+
+### C++ 依賴項
+
+| 依賴      | 說明                                          |
+|-----------|----------------------------------------------|
+| `lsplt`   | Android 的簡單 PLT（程式連結表） 勾取            |
```
Comment (ThePedroo) [inline READMEs/README_zh-TW.md:33]: Can you just remove one space here? Then it will be the best that can be done.

File: READMEs/README_zh-TW.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+
+| 工具            | 說明                                   |
+|-----------------|---------------------------------------|
+| `Android NDK`   | Android 原生開發工具包                  |
+
+### C++ 依賴項
+
+| 依賴     | 說明                                          |
+|----------|----------------------------------------------|
+| `lsplt`  | Android 的簡單 PLT（程式連結表） 勾取            |
```
Comment (ThePedroo) [inline READMEs/README_zh-TW.md:33]:
```suggestion
| `lsplt`  | Android 的簡單 PLT（程式連結表） 勾取           |
```

Oh, I mean this.

Comment (ThePedroo) [reply READMEs/README_zh-TW.md:33]: Please revert the last commit if possible: git reset --soft HEAD~1 and unstage the changes (the command to do so shows in git status)

**Comment by ThePedroo** [comment]
Thank you!

**Pull request merged**